src/lib.rs

```diff
@@ -256,6 +256,17 @@ pub fn outro_cancel(message: impl Display) -> io::Result<()> {
     )
 }
 
+/// Prints a footer of the prompt sequence with a note style.
+/// TODO: Support different styles, i.e. info, warning, error, etc.?
```

Comment:
No need to leave `TODO`: either give up styles, or do it with styles right now :) You need

```rust
enum PromptStyle {
  Step,
  Info,
  ... like in `log`
}
```
I propose to not over-customize, because `cliclack` is supposed to be **simple**.

Reply:
The `TODO` was mostly to catch your attention :) I don't need this customization personally, but I can imagine that users might want to bring specific attention to some information in such a note block using info/warn/error styling. I don't think that this breaks simplicity, the default can be default styling, but being able to specify `.as_warning()` or so would be useful I think in cases where the note is actually important vs. just information.

Reply:
Skipping this for now